libpod/boltdb_state.go

```diff
@@ -705,7 +705,11 @@ func (s *BoltState) AllContainers() ([]*Container, error) {
 				// We just won't include the container in the
 				// results.
 				if errors.Cause(err) != ErrNSMismatch {
-					return err
+					// Even if it's not an NS mismatch, it's
+					// not worth erroring over.
+					// If we do, a single bad container JSON
+					// could render libpod unusable.
+					logrus.Errorf("Error retrieving a container from the databse: %v", err)
```

Comment:
probably ought to include at least the container id (id) in the message here and below.

Reply:
Good thought, done